amp.php

```diff
@@ -5,7 +5,7 @@
  * Plugin URI: https://github.com/automattic/amp-wp
  * Author: WordPress.com VIP, XWP, Google, and contributors
  * Author URI: https://github.com/Automattic/amp-wp/graphs/contributors
- * Version: 1.0-beta4
+ * Version: 1.0-rc1
```

Comment:
Let's name it `1.0-RC1` following WP conventions.

Reply:
Sure, https://github.com/Automattic/amp-wp/pull/1482/commits/17f9eaa8a9d3f1f394d3a143adfe330a0375000a changes is to `1.0-RC1`  both above and in:
```php
define( 'AMP__VERSION', '1.0-RC1' );
```